build.gradle

```
@@ -0,0 +1,57 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+group = 'com.destroystokyo.paper'
+version = '1.14.4-R0.1-SNAPSHOT'
+
+description = """Akarin"""
+
+sourceCompatibility = 1.5
+targetCompatibility = 1.5
```

```suggestion
sourceCompatibility = 1.8
targetCompatibility = 1.8
```
Minecraft uses Java 8